dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Complete.cshtml

```diff
@@ -53,7 +56,7 @@
 else if (Model.FirstTimeUser)
 {
     <govuk-panel class="app-panel--interruption" data-testid="first-time-user-content">
-      <govuk-panel-title>Continue with your NPQ registration</govuk-panel-title>
+      <govuk-panel-title data-testid="first-time-user-postsigninmessage">@postSignInMessage</govuk-panel-title>
```

Comment:
`postSignInMessage` should be title-cased here

Reply:
Wrapping `postSignInMessage` in a `<span>` and adding `data-testid` to that is probably easier